multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/MtaDescriptorPropertiesResolver.java

```diff
@@ -118,6 +125,22 @@ private DeploymentDescriptor correctEntityNames(DeploymentDescriptor descriptor)
                       .validate();
     }
 
+    private void replaceNullLiveRoutesWithEmpty(DeploymentDescriptor descriptor) {
+        for (Module module : descriptor.getModules()) {
+            Map<String, Object> moduleParameters = module.getParameters();
+
+            List<Map<String, Object>> routes = RoutesValidator.applyRoutesType(moduleParameters.get(SupportedParameters.ROUTES));
+
+            for (Map<String, Object> routeMap : routes) {
+                Object liveRoute = routeMap.get(SupportedParameters.LIVE_ROUTE);
+
+                if (Objects.isNull(liveRoute)) {
```

Comment:
when is liveRoute not going to be null here? If we need to initialise it (since it is essentially not a user input parameter, shouldn't we just do it at the start of some specific step, rather than check here if it is null?
Also, what is the value of an empty live route here? What if the user is referencing the live route? One other thing - I assume that the live route should be the same as the normal route if we're not performing a blue-green deploy.